specs/_features/eip7805/validator.md

```diff
@@ -85,39 +123,89 @@ def get_inclusion_committee_assignment(
     return None
 ```
 
-### Lookahead
+#### Lookahead
 
 `get_inclusion_committee_assignment` should be called at the start of each epoch
 to get the assignment for the next epoch (`current_epoch + 1`). A validator
-should plan for future assignments by noting their assigned ILC slot.
+should plan for future assignments by noting their assigned inclusion list
+committee slot.
 
-## New proposer duty
+### Block and sidecar proposal
 
-### Block proposal
+#### Constructing the `BeaconBlockBody`
 
-Proposers are still expected to propose `SignedBeaconBlock` at the beginning of
-any slot during which `is_proposer(state, validator_index)` returns true. The
-mechanism to prepare this beacon block and related sidecars differs from
-previous forks as follows:
+##### ExecutionPayload
 
-#### Update execution client with inclusion lists
+`prepare_execution_payload` is updated from the Electra specs.
 
-The proposer should call `engine_updateInclusionListV1` at
-`PROPOSER_INCLUSION_LIST_CUT_OFF` into the slot with the list of the inclusion
-lists that gathered up to `PROPOSER_INCLUSION_LIST_CUT_OFF`.
+*Note*: In this section, `state` is the state of the slot for the block proposal
+_without_ the block yet applied. That is, `state` is the `previous_state`
+processed through any empty slots up to the assigned slot using
+`process_slots(previous_state, slot)`.
 
-## New inclusion list committee duty
+*Note*: The only change to `prepare_execution_payload` is to call
+`get_inclusion_list_store()` and `get_inclusion_list_transactions()` to set the
```

Comment:
We don't usually include the `()` to indicate a function in the specs. Personally, I do the same though 😢
```suggestion
`get_inclusion_list_store` and `get_inclusion_list_transactions` to set the
```